### ISSUE TITLE
"Multiple paths detected" error on JuliaPro

### ISSUE THREAD

**Issue by yha** [timholy/Revise.jl#68]
With a fresh installation of JuliaPro 0.6.1.1 on Ubuntu 16.04.3:
```
julia> Pkg.add("Revise")
INFO: Cloning cache of Revise from https://github.com/timholy/Revise.jl.git
INFO: Installing Revise v0.1.0
INFO: Package database updated
INFO: METADATA is out-of-date — you may not have the latest version of Revise
INFO: Use `Pkg.update()` to get the latest versions of your packages

julia> using Revise
INFO: Recompiling stale cache file /home/yuval/.juliapro-0.6.1.1/lib/v0.6/Revise.ji for module Revise.
 
julia> using Compat

ERROR (unhandled task failure): Multiple paths detected: String["/home/yuval/.juliapro-0.6.1.1/lib/v0.6/Compat.ji", "/home/yuval/juliapro/JuliaPro-0.6.1.1/JuliaPro/pkgs-0.6.1.1/lib/v0.6/Compat.ji"]
Stacktrace:
 [1] parse_pkg_files(::Symbol) at /home/yuval/juliapro/JuliaPro-0.6.1.1/JuliaPro/pkgs-0.6.1.1/v0.6/Revise/src/Revise.jl:297
 [2] watch_package_impl at /home/yuval/juliapro/JuliaPro-0.6.1.1/JuliaPro/pkgs-0.6.1.1/v0.6/Revise/src/Revise.jl:571 [inlined]
 [3] (::Revise.##9#10{Symbol})() at ./event.jl:73
```
This seems to happen on `import`ing or `using` any built-in package after `using Revise`.
The behavior is the same after running `Pkg.update()` (which updates Revise to version 0.1.1) and restarting Julia.

**Comment by timholy** [comment]
I've not used JuliaPro myself, but this indicates that you've got multiple directories storing precompile files. I added this check to be conservative. Does everything seem to work OK if you comment out [this line](https://github.com/timholy/Revise.jl/blob/90d49fa4c813e37e03039afe016d9ab183a8b5d6/src/Revise.jl#L299)?

**Comment by yha** [comment]
After commenting out the that line, there is no error on the `using` statement.
I tried testing it with a change to the `Distributions` package:

```
julia> using Revise

julia> using Distributions

julia> d = Normal()
Distributions.Normal{Float64}(μ=0.0, σ=1.0)

julia> # editing the rand(Normal) method in JuliaPro/pkgs-0.6.1.1/v0.6/Distributions/src/univariate/continuous/normal.jl

julia> rand(d)
Revise is currently tracking the following files: String[ 
```
_<long list of .jl files>_
```
]
ERROR (unhandled task failure): /home/yuval/juliapro/JuliaPro-0.6.1.1/JuliaPro/pkgs-0.6.1.1/v0.6/Distributions/src/univariate/continuous/normal.jl is not currently being tracked.
Stacktrace:
 [1] revise_file_now(::String) at /home/yuval/juliapro/JuliaPro-0.6.1.1/JuliaPro/pkgs-0.6.1.1/v0.6/Revise/src/Revise.jl:611
 [2] revise() at /home/yuval/juliapro/JuliaPro-0.6.1.1/JuliaPro/pkgs-0.6.1.1/v0.6/Revise/src/Revise.jl:635
 [3] macro expansion at /home/yuval/juliapro/JuliaPro-0.6.1.1/JuliaPro/pkgs-0.6.1.1/v0.6/Revise/src/Revise.jl:774 [inlined]
 [4] (::Revise.##17#18{Base.REPL.REPLBackend})() at ./event.jl:73
```
After this, Julia is not responsive.

**Comment by timholy** [comment]
Interesting. I'm almost certain this is due to having multiple paths and is likely fixed on master (which only runs with Julia 0.7). Is there a JuliaPro for 0.7 yet?

**Comment by yha** [comment]
It seems that there's no JuliaPro for 0.7 yet.
I just tried the newer JuliaPro 0.6.2, and the "multiple paths" error is gone. However, the test above changing the built-in Distributions package still fails in the same way.

**Comment by timholy** [comment]
I am pretty sure this was fixed by the combination of #133, #144, #145. If this is not working, please see https://timholy.github.io/Revise.jl/latest/config.html#Fixing-a-broken-or-partially-working-installation-1.
